service/ui.go

```diff
@@ -275,6 +278,10 @@ func (uis *UIServer) GetServiceApp() *gimlet.APIApp {
 		))
 	}
 
+	// GraphQL
+	app.AddRoute("/graphql").Handler(playground.Handler("GraphQL playground", "/graphql/query")).Options().Post().Get()
```

Comment:
Should this just be GET? What's the behavior if you try to POST here?

Reply:
gql uses POST under the hood to deliver query string in request body. it does some additional stuff too that required adding the extra methods, i.e. Options and Get. otherwise it does not work

Reply:
Right, but do you need to be able to POST to the playground URL? I tried a POST request to that and got the same thing as a GET.

Reply:
interesting, using just the `Get()` method for the playground url does indeed work. i'm curious, what made you notice this?

Reply:
It came about when I was trying to understand what the new routes did. This one provides the GQL playground UI, which it seemed unnecessary to have a POST handler for